osdk/src/commands/run.rs

```diff
@@ -60,6 +60,18 @@ pub fn execute_run_command(config: &RunConfig) {
                     .collect();
                 let mut manifest = config.manifest.clone();
                 manifest.qemu.args.extend(qemu_gdb_args);
+
+                // FIXME: Disable KVM from QEMU args in debug mode.
+                // Currently, the QEMU GDB server does not work properly with KVM enabled.
+                let args_num = manifest.qemu.args.len();
+                manifest.qemu.args.retain(|x| !x.contains("enable-kvm"));
+                if manifest.qemu.args.len() != args_num {
```

Comment:
QEMU provides multiple options to enable kvm. From [the doc, 4.1 Enabling KVM](https://wiki.archlinux.org/title/QEMU), we have at least three methods to eanble kvm
- `accel=kvm` of the `-machine` option 
- `-enable-kvm`
- `-accel kvm`

So I suggest you match pattern `kvm` instead of `enable-kvm`. 
You can remove kvm argument for case 2 (and may be case 3), and just report error and exit for other cases.